workflows/geomxng/geomxngs_fastq_to_dcc.wdl

```diff
@@ -0,0 +1,119 @@
+version 1.0
+
+workflow geomxngs_fastq_to_dcc {
+    input {
+        File ini
+        String fastq_directory
+        String output_directory
+        File? fastq_rename
+        String docker_registry
```

Comment:
Can you replace this with `String docker_registry = "gcr.io/broad-cumulus"` and push an image to `gcr.io/broad-cumulus`?